subjects/change.en.md

```diff
@@ -3,20 +3,21 @@
 ### Instructions
 
 Create 2 functions:
-- `get`: a function that takes a key and return the corresponding
+- `get`: a function that takes a key and returns the corresponding
```

Comment:
I don't believe `return` ever takes an `s` before `the`.
same for update, please don't fix what's not broken haha.

Other changes were made with batch fixes.